src/client/app/effects/snippet.ts

```diff
@@ -224,7 +224,7 @@ export class SnippetEffects {
             }
             AI.trackEvent('Open in playground initiated', { id: correlationId });
             let filename = `script-lab-playground-${environment.current.host}${extension}`;
-            let url = environment.current.config.runnerUrl + `/open-in-playground/${correlationId}/${environment.current.host}/${type}/${id}/${filename}`;
+            let url = environment.current.config.runnerUrl + `/open/${type}/${environment.current.host}/${id}/${filename}?correlationId=${correlationId}`;
```

Comment:
Use the string interopolation here instead of concatenation
let url = `${environment.current.config.runnerUr}/open/${type}/${environment.current.host}/${id}/${filename}?correlationId=${correlationId}`